.github/packaging/project/gha-main.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+DISTRO="$1"
+env
```

Comment:
Is this for debugging? Do we know it this may dump secrets to the log?

Reply:
@kportertx I think the secrets are redacted by value in the logs